Assets/Expansion2/Replacements/citystates.xml

```diff
@@ -227,7 +227,7 @@
                 <Label ID="SecondHighestLabel" Anchor="L,T" Offset="3,0" Style="FontNormal12" />
                 <Label ID="SuzerainLabel" Anchor="L,B" Offset="32,14" Style="FontNormal12" String="LOC_CITY_STATES_SUZERAIN" />
                 <Label ID="Suzerain" Anchor="L,B" Offset="32,2" Style="FontNormal12" String="LOC_CITY_STATES_NONE" TruncateWidth="70" />
```

Comment:
I was actually suggesting using 60 for Suzerain as well. In my second screenshot for the city state of Ngazaramu you can see the label for Poundmaker overlaps with the number of envoys (13). With 60 that overlap goes away. Did you notice any issues when using 60 for the Suzerain label?

```suggestion
                <Label ID="Suzerain" Anchor="L,B" Offset="32,2" Style="FontNormal12" String="LOC_CITY_STATES_NONE" TruncateWidth="60" />
```

Reply:
I didn't see any issues with 60, didn't have a game up where I could see the 70 was an issue.  So, 60 is fine :)